packages/request/src/-private/context.ts

```diff
@@ -159,8 +159,8 @@ export class Context {
   #owner: ContextOwner;
   declare request: ImmutableRequestInfo;
   declare id: number;
-  private declare _isCacheHandler: boolean;
-  private declare _finalized: boolean;
+  declare private _isCacheHandler: boolean;
+  declare private _finalized: boolean;
```

Comment:
as far as I can tell TS doesn't specify the order on this nor care https://tsplay.dev/WJKaRw

declare-first does seem better though

Reply:
lint:fix did this